test/test_ngraph_exec.cpp

```diff
@@ -0,0 +1,499 @@
+  Graph input_graph(OpRegistry::Global());
+  ASSERT_OK(LoadGraph("test_axpy_int8_launchop.pbtxt", &input_graph));
+
+  // Create the inputs for this graph
+  Tensor x(DT_INT8, TensorShape({2, 2}));
+  Tensor y(DT_INT8, TensorShape({2, 2}));
+
+  std::vector<TensorShape> input_shapes;
+  input_shapes.push_back(x.shape());
+  input_shapes.push_back(y.shape());
+
+  shared_ptr<ng::Function> ng_function;
+  ASSERT_OK(TranslateTFGraphNoStatic(input_shapes, input_graph, ng_function));
+
+  // Create the nGraph backend
+  auto backend = ng::runtime::Backend::create("CPU");
+
+  // Allocate tensors for arguments a, b, c
+  ng::Shape ng_shape_x(x.shape().dims());
+  for (int i = 0; i < x.shape().dims(); ++i) {
+    ng_shape_x[i] = x.shape().dim_size(i);
+  }
+
+  ng::Shape ng_shape_y(y.shape().dims());
+  for (int i = 0; i < y.shape().dims(); ++i) {
+    ng_shape_y[i] = y.shape().dim_size(i);
+  }
+
+  auto t_x = backend->create_tensor(ng::element::i8, ng_shape_x);
+  int8 v_x[2][2] = {{1, 1}, {1, 1}};
+  t_x->write(&v_x, 0, sizeof(v_x));
+
+  auto t_y = backend->create_tensor(ng::element::i8, ng_shape_y);
+  t_y->write(&v_x, 0, sizeof(v_x));
+
+  // Allocate tensor for the result(s)
+  vector<shared_ptr<ng::runtime::Tensor>> outputs;
+  for (size_t i = 0; i < ng_function->get_output_size(); i++) {
+    auto shape = ng_function->get_output_shape(i);
+    auto elem_type = ng_function->get_output_element_type(i);
+    auto t_result = backend->create_tensor(elem_type, shape);
+    outputs.push_back(t_result);
+  }
+
+  // Execute the nGraph function.
+  cout << "Calling nGraph function\n";
+  auto exec = backend->compile(ng_function);
+  exec->call(outputs, {t_x, t_y});
+
+  for (size_t i = 0; i < ng_function->get_output_size(); i++) {
+    DumpNGTensor<int8>(cout, ng_function->get_output_op(i)->get_name(),
+                       outputs[i]);
+    cout << endl;
+  }
+  // Add the validation logic
+  // TODO
+}
+
+TEST_F(NGraphExecTest, MixedTensors) {
+  Graph input_graph(OpRegistry::Global());
+  ASSERT_OK(LoadGraph("test_axpy_launchop.pbtxt", &input_graph));
+
+  // Create the inputs for this graph
+  DataType tf_dt = DT_FLOAT;
+  TensorShape tf_shape = TensorShape({2, 3});
+  int num_inputs = 2;
+
+  // Run Graph on TF: Expected output
+  Tensor x(tf_dt, tf_shape);
+  Tensor y(tf_dt, tf_shape);
+  AssignInputValues(x, 1.0f);
+  AssignInputValues(y, 1.0f);
+  std::vector<Tensor> tf_inputs = {x, y};
+  vector<Tensor> expected_outputs;
+  vector<pair<string, Tensor>> feed_dict = {{"x", x}, {"y", y}};
+  vector<string> out_node_names = {"add", "mul"};
+  ASSERT_OK(
+      RunGraphOnTF(input_graph, feed_dict, out_node_names, expected_outputs));
+
+  // Run on nGraph
+  // Translate Graph
+  std::vector<TensorShape> tf_input_shapes(num_inputs, tf_shape);
+  shared_ptr<ng::Function> ng_function;
+  ASSERT_OK(
+      TranslateTFGraphNoStatic(tf_input_shapes, input_graph, ng_function));
+
+  // Create the nGraph backend
+  string backend_name = "INTERPRETER";
```

Comment:
I noticed that we are are testing a lot of our tests with Interpreter as backend,
do you think it would be good idea to get backend in user input, through environment variable 
or flag, and in case input not provided Interpreter will be default backend.

Reply:
OverrideBackendFromEnv(&backend_name) this does that